### PR TITLE
Update deps. Adjust configs to API changes in ICE

### PR DESCRIPTION
**Pull request by mickel8**

### CODE REVIEW

File: webrtc/videoroom/config/ice.exs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import Config
```
Comment (mickel8) [inline webrtc/videoroom/config/ice.exs:1]: I am not sure if this should be a separate file. Now we have so many config files

Comment (Qizot) [reply webrtc/videoroom/config/ice.exs:1]: I'm pretty sure we should not add config files that are not default to elixir ones (I mean the names). If this is hard coded then it should probably be put in `config/config.exs` but I guess we would like to provide stun servers in runtime.

Comment (Qizot) [reply webrtc/videoroom/config/ice.exs:1]: But I have no idea what should be the format for turn servers as they require username and password.

**Review by Qizot — CHANGES_REQUESTED** [review]

File: webrtc/videoroom/config/ice.exs (hunk 1 above)
Comment (mat-hek) [reply webrtc/videoroom/config/ice.exs:1]: why can't we just put that in config.exs or runtime.exs?

**Review by Qizot — APPROVED** [review]

**Pull request merged**